spec/models/dish_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe Dish, type: :model do
+
+  describe "DB tables" do
+    it { is_expected.to have_db_column :name }
+    it { is_expected.to have_db_column :description }
+    it { is_expected.to have_db_column :price }
+    it { is_expected.to have_db_column :ready_time }
```

Comment:
You need to add a migration to rename that attribute in order to make these tests to pass.